docs/guides/multiple-documents.md

```diff
@@ -0,0 +1,107 @@
+# Serving multiple OAS documents
+
+A single Prism instance serves one OAS document.
+In order to serve multiple documents, you can use `docker-compose` utility.
+
+## Preparing configuration files
+
+Depending on your needs, you might have a `docker-compose.yaml` file like this:
+
+```yaml
+version: "3"
+services:
+  proxy:
+    image: nginx:alpine
+    volumes:
+      - ./nginx.conf:/etc/nginx/nginx.conf
+    ports:
+      - "8080:80"
+    depends_on:
+      - prism_1
+      - prism_2
+  prism_1:
+    image: stoplight/prism:3
+    command: mock -p 4010 --host 0.0.0.0 https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v2.0/yaml/petstore.yaml
```

Comment:
Thes urls are too long. Any idea what we could do here?

Reply:
@XVincentX, we could go back to envs, no other ideas

Reply:
Let's not worry about this